clang/test/CodeGen/aarch64-sve2p1-intrinsics/acle_sve2p1_fclamp.c

```diff
@@ -11,10 +11,16 @@
 // RUN: %clang_cc1 -fclang-abi-compat=latest -triple aarch64-none-linux-gnu -target-feature +sve2p1 \
 // RUN:   -S -disable-O0-optnone -Werror -Wall -o /dev/null %s
 // RUN: %clang_cc1 -fclang-abi-compat=latest -triple aarch64-none-linux-gnu -target-feature +sme2 -target-feature +sve \
-// RUN:   -S -disable-O0-optnone -Werror -Wall -o /dev/null %s
+// RUN:   -S -DTEST_SME2 -disable-O0-optnone -Werror -Wall -o /dev/null %s
 
 #include <arm_sve.h>
 
+#ifndef TEST_SME2
+#define ATTR
+#else
+#define ATTR __arm_streaming
```

Comment:
I think this should be __arm_streaming_compatible?

Reply:
arm_sve.td has SVFCLAMP as `IsStreamingOrSVE2p1` so I think it has to be `__arm_streaming``, or is the builtin def incorrect?

Reply:
Sorry, this was my mistake! You were right to add it as `__arm_streaming` :)